pay-api/src/pay_api/models/fee_schedule.py

```diff
@@ -91,6 +92,16 @@ def find_all(cls, corp_type_code: str = None, filing_type_code: str = None):
         if corp_type_code:
             query = query.filter_by(corp_type_code=corp_type_code)
 
+        if description:
+            # description is free text search
+            description_list = description.replace(' ', '%')
```

Comment:
may be have a better name than `_list`, I thought it's a list and was wondering how `lower()' function would work on list. Then realized it's not list but a string.

Reply:
changed